core/unit_test/TestGraph.hpp

```diff
@@ -739,4 +739,61 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), end_of_submit_control_flow) {
             value_A + 2 * value_B + value_C + value_D + value_E + value_F);
 }
 
+template <typename Exec>
+struct GraphNodeTypes {
+  //! Type of a kernel node built using a Kokkos parallel construct.
```

Comment:
The doxygen markup is not needed. Same for the other one.

Reply:
OK, done.

Reply:
(Just noting that it still shows up like a `\\!` Missing a push?)

Reply:
Indeed, and done.